javascript/javascript3/week2/resolveAfter.html

```diff
@@ -0,0 +1,47 @@
+<head>
+    <title>HyfBay</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+
+</head>
+
+<body>
+    <main>
+
+
+    </main>
+
+    <script>
+        /*
+                                function myFunction(resolveAfter) {
+
+                                    return new Promise((resolve, reject) => {
+                                        if (resolveAfter != "") {
+                                            setTimeout(() => {
+                                                resolve();
+                                            }, resolveAfter * 1000)
+                                        } else {
+                                            reject("Failure!");
+                                        }
+                                    })
+
+                                }
+
+                                myFunction(8).then(() => {
+                                    console.log(`I am called asynchronously`); // logged out after 8 seconds
+                                });*/
```

Comment:
don't leave long comments in the code. git keeps track of your changes so you can get back the older version of the file any time you want.